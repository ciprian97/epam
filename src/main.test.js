import { render } from "@testing-library/react";
import renderer from "react-test-renderer";

import Main from "./main";
describe("Main component", () => {
  it("should render correctly ", async () => {
    const { container } = render(<Main />);

    expect(container).toBeTruthy();
  });

  it("should render correctly", () => {
    const tree = renderer.create(<Main />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
