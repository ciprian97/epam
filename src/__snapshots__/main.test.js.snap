// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should render correctly 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": 200,
      "width": "30%",
    }
  }
>
  <div
    className="ui input"
    style={
      Object {
        "width": "400px",
      }
    }
  >
    <input
      defaultValue=""
      onChange={[Function]}
      placeholder="Search for a movie..."
      type="text"
    />
  </div>
  <p>
    No results
  </p>
</div>
`;
